Sources/StreamChatUI/ChatMessageList/RETHINK/MessageListVC.swift

```diff
@@ -392,6 +409,63 @@ class MessageListVC<ExtraData: ExtraDataTypes>: _ViewController, UICollectionVie
             reactionsController: reactionsController
         )
     }
+
+    private func restartUploading(for attachment: ChatMessageDefaultAttachment) {
```

Comment:
Why `private`?

Reply:
hmm, didn't expect this to be changed/overriden in client code 🤔 At the same time we are `open by default` so let's update the access level